scripts/import_measurements.py

```diff
@@ -1,18 +1,183 @@
-from db import get_session
+import csv
+import functools
+from io import StringIO
 from pathlib import Path
 
 import sqlalchemy as sql
 
-from models import Measurement
+from db import get_session
+from lib.db import execute_text
+from models import (
+    Measurement,
+    Bioreplicate,
+    Strain,
+)
+
+# Note: insert methods taken from Measurement, work on legacy-format data
```

Comment:
This file is just here to maintain the ability to import the previous format of data files. The code should be removed at some point, but it will be a good idea for me to make sure that the studies we do have are portable in the new data format (columns, etc).